src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
```

Comment:
remove static modifier 
```suggestion
    public String getRandomColor() {
```